src/run/port.rs

```diff
@@ -65,6 +68,14 @@ bi_enum! {
     /// storing the targets of the wires connected to the two auxiliary ports of
     /// this node.
     Op = 4,
+    /// An `F32` port represents the principal port of an 32-bit floating
+    /// point node.
+    ///
+    /// The top 32 bits of the port are the value of this node and is
+    /// accessible with [`Port::float`].
+    ///
+    /// The 4th bit from the bottom is currently unused in this port.
+    F32 = 5,
```

Comment:
this should become `4` and `Op` should become `5`, that way `> F32` checks for nodes with allocations